docs/openapi/openapi.yaml

```diff
@@ -302,13 +357,43 @@ components:
       allOf:
         - $ref: "#/components/schemas/Person"
       type: object
+      required:
+        - dateOfBirth
+        - email
+        - status
       properties:
+        dateOfBirth:
+          type: string
+          format: date
+        email:
+          type: string
+          x-faker: internet.email
         status:
           type: string
           enum:
             - active
             - replacement
             - removed
+    CertificateProvider:
+      allOf:
+        - $ref: "#/components/schemas/Person"
+      type: object
+      required:
+        - email
+        - channel
+      properties:
+        email:
+          type: string
+          x-faker: internet.email
+        channel:
```

Comment:
Same comment again: could we link this to a glossary so we know what each "channel" meant at the time we set a value for that property? What exactly is meant by "paper" or "online"? There could be other media in the future not covered by these, and these might even potentially become meaningless 50 years from now.